array_api_compat/_dask_ci_shim.py

```diff
@@ -0,0 +1,13 @@
+"""
+A little CI shim for the dask backend that
+disables the dask scheduler
+"""
+import dask
+dask.config.set(scheduler='synchronous')
```

Comment:
I don't know if this is needed anymore (it didn't seem to help with the test that OOMed - which is now disabled).

Leaving this in just in case I haven't found the correct options.